roles/qe-server/tasks/main.yml

```diff
@@ -49,6 +49,37 @@
    - rh-python35-python-jinja2
 
 #
+# support for web tests
+#
+
+- name: Install packages required for web tests
+  yum: name={{ item }} state=present enablerepo=epel
+  with_items:
+    - xorg-x11-server-Xvfb
+    - x11vnc
+    - firefox
+
+- name: Install systemd service unit file
+  copy:
+    src={{ item }}
+    dest=/etc/systemd/system/{{ item }}
+    mode=0664
+    owner=root
+    group=root
+    setype=systemd_unit_file_t
+  with_items:
+    - Xvfb@.service
+    - x11vnc@.service
+
+- name: Open port for x11vnc server via firewalld
+  firewalld:
+    port=5900/tcp
```

Comment:
Not sure, how to best solve the various port numbers here so I would maybe open range of ports ``5900 - 5920`` (or more/less, not sure how many opened displays are likely possible).